Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -600,11 +600,13 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
                             try
                             {
                                 Message message = _messageCodec.Decode(copied, false);
+                                Guid reqId = new Guid(copied[0].ToByteArray());
```

Comment:
Ah... we can't assume the first frame can be translated as GUID always, so need handling for that. 😢

Reply:
Added a fallback exception If `Identity` cannot be parsed as GUID, create a new one.